client/src/app/primitives/modal/Modal.js

```diff
@@ -70,4 +77,60 @@ Modal.defaultProps = {
   onClose: () => {}
 };
 
-export default Modal;
+Modal.Header = Header;
+
+Modal.Body = Body;
+
+Modal.Title = Title;
+
+Modal.Close = Close;
+
+Modal.Footer = Footer;
+
+
+function Title(props) {
+
+  return (
+    <h2 className="modal-title">
+      { props.children }
+    </h2>
+  );
+}
+
+function Close(props) {
```

Comment:
Do we want to let the plugin authors to decide where they want to put the X button? In this case, the default CSS might be confusing since the position of the button is different whether it's
```javascript
<Modal>
  <Modal.Close />
</Modal>

// or

<Modal>
  <Modal.Header>
     <Modal.Close />
  </Modal.Header>
</Modal>
```

Reply:
Fixed via https://github.com/camunda/camunda-modeler/pull/1568#discussion_r341482736

Reply:
>  prop to disable the button

Should always be enabled. There should be no situation where people are forced to keep a modal open. ESC / CLOSE should always be an available option. That is how modals are supposed to work.